rbac/apps.py

```
@@ -0,0 +1,19 @@
+"""
+تنظیمات اپلیکیشن RBAC (Role-Based Access Control)
+"""
+from django.apps import AppConfig
+
+
+class RbacConfig(AppConfig):
+    """پیکربندی اپلیکیشن RBAC"""
+    
+    default_auto_field = 'django.db.models.BigAutoField'
+    name = 'rbac'
+    verbose_name = 'مدیریت نقش‌ها و دسترسی‌ها'
```

Ruff (0.12.2)

String contains ambiguous `ه` (ARABIC LETTER HEH). Did you mean `o` (LATIN SMALL LETTER O)?

(RUF001)